README.md

```diff
@@ -100,7 +100,8 @@ To download the CloudFormation template to deploy on your own, for example by [u
 
 https://s3.amazonaws.com/solution-builders-us-east-1/amazon-cloudfront-secure-static-site/latest/main.yaml
 
-## Update the website content locally before deploying the solution
+## Making Changes to the Solution

```

Comment:
```suggestion
## Customizing the Solution
```

Suggestion only, but I think this fits better as you're not really changing "the solution".